test/plugin/constants.rb

```diff
@@ -477,4 +563,50 @@ module Constants
     '' => '_'
   }
   ALL_TAGS = VALID_TAGS.merge(INVALID_TAGS)
+
+  # Stub value for Monitored resources from Metadata Agent.
+  # Map from the local_resource_id to the retrieved monitored resource.
+  MONITORED_RESOURCE_STUBS = {
+    # Implicit GCE instance.
+    IMPLICIT_LOCAL_RESOURCE_ID =>
+      {
+        'type' => COMPUTE_CONSTANTS[:resource_type],
+        'labels' => {
+          'zone' => METADATA_ZONE,
+          'instance_id' => METADATA_VM_ID
+        }
+      }.to_json,
+    # Docker container stderr g stdout logs.
```

Comment:
I think you meant to keep the `/` here instead of `g`.

Reply:
LOL. Oops. Changed.